cli.js

```diff
@@ -145,21 +160,30 @@ function exit(results) {
 	var failed = sum(stats, 'failCount');
 
 	log.write();
-	log.report(passed, failed);
+	log.report(passed, failed, unhandledRejectionCount, uncaughtExceptionCount);
 	log.write();
 
 	if (failed > 0) {
 		log.errors(flatten(tests));
 	}
 
+	process.stdout.write('');
+
+	flushIoAndExit(
+		failed > 0 || unhandledRejectionCount > 0 || uncaughtExceptionCount > 0 ? 1 : 0
+	);
+}
+
+function flushIoAndExit(code) {
 	// TODO: figure out why this needs to be here to
 	// correctly flush the output when multiple test files
 	process.stdout.write('');
+	process.stderr.write('');
```

Comment:
oops - I meant to delete that other one.

There should be one each `stderr` and `stdout`. This belongs in the "it seamed to help" pile. Definitely room to experiment with removing it down the road.